src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -165,6 +166,17 @@ public String getDurationString() throws AbortException {
         return build().getDurationString();
     }
 
+    @Whitelisted
+    public String getLog() throws AbortException {
+        StringWriter writer = new StringWriter();
+        try {
+            build().getLogText().writeLogTo(0, writer);
+            return writer.toString();
+        } catch (IOException e) {
+            throw new AbortException("Could not read log for build " + externalizableId + ": " + e);
```

Comment:
This discards the original stack trace, and is really unnecessary. Just let the method throw up `IOException`.